### PR TITLE
Added lastUpdated in API + Added Last Updated in UI

### PR DESCRIPTION
**Pull request by kquirapas**
Changes to address #38:
1. [lastUpdated in Response JSON](https://github.com/web3phl/directory-api/commit/0725705542db7ae1d78a1dab6065768cff378f22)
2. [Last Updated in UI](https://github.com/web3phl/directory/pull/61/commits/2c839eb5919c9dfe974badb66611d0ac981fcfd4)
3. [Fix directories undefined error](https://github.com/web3phl/directory/pull/61/commits/f9b57ce2ad546da6b3b05b1c842f6cd72846e41e)

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -46,9 +48,17 @@ const Home: NextPage = () => {
           Philippines. 📚💜💻{" "}
         </p>
 
+        {!!directories && (
+          <p className="mb-[20px] text-center text-base leading-6 text-neutral-light md:mb-[50px] md:text-left">
```
Comment (jnebab) [inline src/pages/index.tsx:52]: can we try a lighter shade for this? need to emphasize the description of the site more than this text

**Review by jnebab — COMMENTED** [review]
lighter shade for last updated text

**Comment by kquirapas** [comment]
@jnebab Made last updated lighter
Also added neutral-lightest in tailwind config

image

**Review by warengonzaga — APPROVED** [review]
LPG 💪

**Pull request merged**